Apps/WebClient/src/Server/Controllers/NoteController.cs

```diff
@@ -139,9 +139,10 @@ public async Task<IActionResult> Get(Guid noteId)
                 return new ForbidResult();
             }
 
-            //RequestResult<Database.Models.Note> result = this.noteService.Get(noteId);
+            // RequestResult<Database.Models.Note> result = this.noteService.Get(noteId);
 
             // TODO: Compare note hdid with current jwt hdid.
+            // TODO: Do we need to implement this?
```

Comment:
I don't see a use case for this but I think we were following the restful standards, we don't need it in my opinion